test/Microsoft.Sbom.Targets.E2E.Tests/GenerateSbomE2ETests.cs

```diff
@@ -111,23 +113,25 @@ private void SetDefaultProperties(Project sampleProject)
         }
     }
 
-    private void RestoreBuildPack(Project sampleProject)
+    private void RestoreBuildPack(Project sampleProject, [CallerMemberName] string callerName = null)
     {
         var logger = new ConsoleLogger();
-
         // Restore the project to create project.assets.json file
-        var restore = sampleProject.Build("Restore", new[] { logger });
+        var restore = sampleProject.Build("Restore", new ILogger[] { GetBinLog(callerName, "Restore"), logger });
         Assert.IsTrue(restore, "Failed to restore the project");
 
         // Next, build the project
-        var build = sampleProject.Build(logger);
+        var build = sampleProject.Build(new ILogger[] { GetBinLog(callerName, "Build"), logger });
         Assert.IsTrue(build, "Failed to build the project");
 
         // Finally, pack the project
-        var pack = sampleProject.Build("Pack", new[] { logger });
+        var pack = sampleProject.Build("Pack", new ILogger[] { GetBinLog(callerName, "Pack"), logger });
         Assert.IsTrue(pack, "Failed to pack the project");
     }
 
+    // binlogs are unique per name, so this ensures distinct names for the different stages
+    private BinaryLogger GetBinLog(string callerName, string target) => new BinaryLogger { Parameters = $"{callerName}.{target}.binlog" };
```

Comment:
I assume these changes are just for debugging purposes and not actually necessary?

Reply:
exactly - needed to see what was failing in the tests in a stable way, and the binlog viewer helps a ton with figuring out _why_ the specific build failed.